index.html

```diff
@@ -249,23 +249,32 @@ <h3>Partnerships</h3>
                             <div class="floe-front-news">
                                 <h3>News</h3>
 
+                                <h4>Introducing UI Options Plus (UIO+)</h4>
+                                <time class="floe-date" datetime="2018-01-31">January 31, 2018</time>
+                                <p>
+                                    User Interface Options also known as Learner Options provides individuals with the ability to customize a web
+                                    site to meet their personal needs and preferences. Currently, to provide such customizations, the web page needs to incorporate the UI Options widget into the site's markup.
+                                    However, not every site will have UI Options or site personalizations incorporated.
+                                </p>
+
+                                <p>
+                                    User Interface Options Plus (UIO+) aims to address this issue by enabling the personalization experience of UI Options as a browser extension.
+                                    Preferences can be set via the UIO+ adjuster panel or, if on a <a href="https://gpii.net/">GPII (Global Public Inclusive Infrastructure)</a> enabled machine, from a keyed in preference set.
+                                </p>
+                                <p><a href="./news/2018-01-31-uioPlus.html">Read more: Introducing UI Options Plus (UIO+)</a></p>
```

Comment:
Instead of two paragraphs, we can condense this into one paragraph:
"User Interface Options Plus (UIO+) aims to address the portability of web content personalization by enabling the experience of UI Options as a browser extension. Preferences can be set via the UIO+ adjuster panel or, if on a <a href="https://gpii.net/">GPII (Global Public Inclusive Infrastructure)</a> enabled machine, from a keyed in preference set.